### ISSUE TITLE
Diff support for tables using fulltext index WITH PARSER

### ISSUE THREAD

**Issue by patdz** [skeema/tengo#13]
```sql
 FULLTEXT KEY `title_index` (`title`) /*!50100 WITH PARSER `ngram` */
```

**Comment by evanelias** [comment]
Thank you for the issue report. I don't think MySQL exposes the parser name anywhere in `information_schema`, so this will require parsing `SHOW CREATE TABLE` output. Converting this issue into a feature request accordingly.

**Comment by evanelias** [comment]
This is now implemented in master, and will be included in the next release (v0.9.3) within the next couple weeks.

**Comment by evanelias** [comment]
Happy to announce that this change has been included in today's release of [Skeema CLI v1.4.3](https://github.com/skeema/skeema/releases/tag/v1.4.3). Additionally, the [SaaS Skeema.io CI service](https://www.skeema.io/ci) now supports use of the `ngram` parser in MySQL 5.7+ and Percona Server 5.7+, as these database versions ship with that parser pre-installed.
